python/cuml/cuml/manifold/umap.pyx

```diff
@@ -510,15 +520,10 @@ class UMAP(UniversalBase,
             else:
                 umap_params.build_algo = graph_build_algo.NN_DESCENT
                 build_kwds = self.build_kwds or {}
-                umap_params.nn_descent_params.graph_degree = <uint64_t> build_kwds.get("nnd_graph_degree", 64)
-                umap_params.nn_descent_params.intermediate_graph_degree = <uint64_t> build_kwds.get("nnd_intermediate_graph_degree", 128)
+                umap_params.nn_descent_params.intermediate_graph_degree = <uint64_t> build_kwds.get("nnd_intermediate_graph_degree", 0)
```

Comment:
Please add a comment here to tell developers that the default is inferred on the C++ side when the parameter is set to 0.